### PR TITLE
Fixes Deaf Listening

### LINKED ISSUE #7332: You can hear tape recordings while deaf
**Problem Description**:
Tape recordings are able to be heard by the deaf.

**What did you expect to happen**: 
By the virtue of deafness, an audio recording should be impossible to hear.

**What happened instead**:
I heard a tape recording while deaf.

**Why is this bad/What are the consequences:**
It doesn't really make sense and it could allow for direct communication with the deaf.

**Steps to reproduce the problem**:
I was flashbanged with no protection. While the deafness lingered, someone started playing a tape recording and I could hear what was being said, as seen here.
http://imgur.com/a/pMEgg

### PR DESCRIPTION
**Pull request by TullyBurnalot**
Fixes #7332

- Deaf people can no longer hear tape recordings

:cl:
bugfix: Deaf people cannot hear tape recordings anymore
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/taperecorder.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	desc = "A device that can record to cassette tapes, and play them. It automatically translates the content in playback."
 	icon_state = "taperecorder_empty"
 	item_state = "analyzer"
+	atom_say_verb = "says"
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/taperecorder.dm]: This is the default verb, so you don't need to define it here.

File: code/game/objects/items/devices/taperecorder.dm [hunk 2]
```diff
@@ -155,8 +156,7 @@
 		return
 	else if(playing)
 		playing = 0
-		var/turf/T = get_turf(src)
-		T.visible_message("<font color=Maroon><B>Tape Recorder</B>: Playback stopped.</font>")
+		atom_say("<font color=Maroon><B>Tape Recorder</B>: Playback stopped.</font>")
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/taperecorder.dm]: You'll want to kill off all the formatting; `atom_say` handles all of that.

File: code/game/objects/items/devices/taperecorder.dm [hunk 3]
```diff
@@ -185,19 +185,18 @@
...
 			break
-		var/turf/T = get_turf(src)
-		T.visible_message("<font color=Maroon><B>Tape Recorder</B>: [mytape.storedinfo[i]]</font>")
+		atom_say("<font color=Maroon><B>Tape Recorder</B>: [mytape.storedinfo[i]]</font>")
 		if(mytape.storedinfo.len < i + 1)
 			playsleepseconds = 1
 			sleep(10)
 			T = get_turf(src)
-			T.visible_message("<font color=Maroon><B>Tape Recorder</B>: End of recording.</font>")
+			atom_say("<font color=Maroon><B>Tape Recorder</B>: End of recording.</font>")
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/taperecorder.dm]: `say("End of recording.")`

File: code/game/objects/items/devices/taperecorder.dm [hunk 4]
```diff
@@ -185,19 +185,18 @@
...
 		else
 			playsleepseconds = mytape.timestamp[i + 1] - mytape.timestamp[i]
 		if(playsleepseconds > 14)
 			sleep(10)
 			T = get_turf(src)
-			T.visible_message("<font color=Maroon><B>Tape Recorder</B>: Skipping [playsleepseconds] seconds of silence</font>")
+			atom_say("<font color=Maroon><B>Tape Recorder</B>: Skipping [playsleepseconds] seconds of silence</font>")
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/taperecorder.dm]: `atom_say("Skipping [playsleepseconds] seconds of silence")`

File: code/game/objects/items/devices/taperecorder.dm [hunk 5]
```diff
@@ -185,19 +185,18 @@
 			break
 		if(mytape.storedinfo.len < i)
 			break
-		var/turf/T = get_turf(src)
-		T.visible_message("<font color=Maroon><B>Tape Recorder</B>: [mytape.storedinfo[i]]</font>")
+		atom_say("<font color=Maroon><B>Tape Recorder</B>: [mytape.storedinfo[i]]</font>")
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/taperecorder.dm]: No formating, just:

`atom_say(mytape.storedinfo[i])` (be sure to test this though).

**Comment by SamHPurp** [comment]
Missing your name from the changelog.

**Comment by TullyBurnalot** [comment]
@Fox-McCloud

**Pull request merged**